### PR TITLE
[CPU] ExperimentalDetectronTopKROIs dynamic done

### PR DESCRIPTION
**Pull request by egorshul**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Comment by egorshul** [comment]
@iefode could you take a look at test part please?

**Review by iefode — APPROVED** [review]

**Comment by egorshul** [comment]
@mandrono could you take a look please?

**Review by mandrono — APPROVED** [review]

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp [hunk 1]
```diff
@@ -16,10 +16,6 @@ using namespace InferenceEngine;
 
 bool MKLDNNExperimentalDetectronTopKROIsNode::isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept {
     try {
-        if (isDynamicNgraphNode(op)) {
```
[the remark this answers is not in the document]
Comment (egorshul) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp:19]: Fixed in acbc8ecd7aa48d20512d1afc937be7276a174b01.

**Comment by egorshul** [comment]
@iefode, @mandrono  thank you!

@dmitry-gorokhov could you take a look please?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp [hunk 2]
```diff
@@ -44,10 +40,10 @@ MKLDNNExperimentalDetectronTopKROIsNode::MKLDNNExperimentalDetectronTopKROIsNode
         IE_THROW() << "Operation with name '" << op->get_friendly_name() <<
             "' is not an instance of ExperimentalDetectronTopKROIs from opset6.";
 
-    if (getOriginalInputsNumber() != 2 || getOriginalOutputsNumber() != 1)
+    if (inputShapes.size() != 2 || outputShapes.size() != 1)
         IE_THROW() << errorPrefix << " has incorrect number of input/output edges!";
 
-    if (op->get_input_shape(INPUT_ROIS).size() != 2 || op->get_input_shape(INPUT_PROBS).size() != 1)
+    if (getInputShapeAtPort(INPUT_ROIS).getDims().size() != 2 || getInputShapeAtPort(INPUT_PROBS).getDims().size() != 1)
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp:46]: minor: getRank()

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp [hunk 3]
```diff
@@ -44,10 +40,10 @@ MKLDNNExperimentalDetectronTopKROIsNode::MKLDNNExperimentalDetectronTopKROIsNode
...
         IE_THROW() << errorPrefix << " has nsupported input shape";
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp:47]: minor: unsupported

**Review by dmitry-gorokhov — APPROVED** [review]
Please fix remaining issue in separate PR

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp (hunk 2 above)
Comment (egorshul) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp:46]: PR with fix [#7885](https://github.com/openvinotoolkit/openvino/pull/7885). Thank you!

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp (hunk 3 above)
Comment (egorshul) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_experimental_detectron_topkrois_node.cpp:47]: PR with fix [#7885](https://github.com/openvinotoolkit/openvino/pull/7885). Thank you!

**Pull request merged**